omero/developers/index.txt

```diff
@@ -115,6 +115,7 @@ Web
     Web/WritingViews
     Web/WritingTemplates
     Web/CSRF
+    Web/Webclient
```

Comment:
I think it make sense to keep it last. We are planning to add more, aren't we? Maybe create separate doctree called Webclient ?